var/spack/repos/builtin/packages/libtool/package.py

```diff
@@ -3,7 +3,7 @@
 class Libtool(Package):
     """libtool -- library building part of autotools"""
     homepage = "https://www.gnu.org/software/libtool/"
-    url      = "http://ftpmirror.gnu.org/libtool/libtool-2.4.2.tar.gz"
+    url      = "http://ftpmirror.gnu.org/libtool/libtool-2.4.6.tar.gz"
```

Comment:
`spack uninstall libtool@2.4.6` and `spac uninstall libtool@2.4.2` and `spack install libtool` works without change to url to fetch 2.4.6 directly. Earlier it fetched 2.4.2 version for some reason. 
----
PS: With a fresh git clone of origin I got the same error as [issue 544](https://github.com/LLNL/spack/issues/544).